first draft for the all in one (#1088)
aneoconsulting/ArmoniK: .docs/content/1.installation/3.aws/aws-all-in-one-deployment.md

# AWS all in one deployment

Deploying on AWS is similar to deploying on localhost but with the necessity to deploy an S3 bucket first.

## Generate S3 bucket key
Execute the following command to generate a prefix key:

```bash

> make bootstrap-deploy PREFIX=PREFIX_NUMBER

Reviewer: ```suggestion
make bootstrap-deploy PREFIX=<PREFIX_KEY>
```

```

To deploy, simply execute the following command:
```bash

> make deploy PREFIX=PREFIX_NUMBER

Reviewer: ```suggestion
make deploy PREFIX=<PREFIX_KEY>
```

```

> Note : after the deployment, you can retrieve the prefix key in the [prefix folder](aws/all/generated/.prefix) 

Reviewer: ```suggestion
Note : after the deployment, you can retrieve the prefix key in the prefix file: `<PATH_TO_AWS_FOLDER>/generated/.prefix`
```

To destroy the deployment, execute the following command: 

```bash

> make destroy PREFIX=PREFIX_NUMBER

Reviewer: ```suggestion
make destroy PREFIX=<PREFIX_KEY>
```

```

To destroy the AWS prefix key, execute the following command: 

```bash

> make bootstrap-destroy PREFIX=PREFIX_NUMBER

Reviewer: ```suggestion
make bootstrap-destroy PREFIX=<PREFIX_KEY>
```
